content-test/components/NewTabPage.story.js

```diff
@@ -34,8 +34,19 @@ const Container = props => (
 // itself.
 
 storiesOf("NewTabPage", module)
-  .add("Be on the lookout popup", () =>
-    <Container>
-      <NewTabPage {...mockData} page="SPOTLIGHT_STORYBOOK" dispatch={dispatch} />,
-    </Container>
+  .add("data available (this.props.isReady==true)", () => {
```

Comment:
nit: Most other items (what's the term? story? view?) start with an uppercase. Also, not sure if the verbose `(this.props.isReady==true)` adds much over the "human" readable label. ;)